app/css/buddyup.css

```diff
@@ -29,6 +31,21 @@ html {
 .no-data {
     margin: 1rem;
 }
+.user-list ul {
+    height: 48px;
+    margin-top: 1rem;
+    overflow: hidden;
+    padding-left: 1rem;
+}
+.user-list li {
+    float: left;
+    margin-right: 1rem;
+}
+.user-list ul img {
+    border-radius: 24px;
```

Comment:
We can keep this at 24px but `border-radius: 50%;` makes it independant of the box size.